src/frontend/src/App.js

```diff
@@ -45,6 +47,20 @@ function App() {
       disable_ocean_mask: formRefs.disableOceanMask.current.checked,
     };
 
+    try {
+      const data = await fetch('/static/taxa_names.json').then((response) => response.json());
```

Comment:
let's put the path to the file with taxa_names in a separate constant. most likely, this path or file name will change at some point, so it will be easier to manage it

Reply:
also I think laoding taxa names into a json and holding it throughout the application could be a good idea instead of fetching each time

Reply:
Maybe, but there are a lot of them. How much memory will they take to hold in memory?

Reply:
1.4 mb I believe, I can revert it back to load each time